yc-k8s-ingress-class/README.md

```diff
@@ -0,0 +1,143 @@
+# ALB Ingress Class
+
+This guide shows how to deploy two ingresses with ingressClass nginx and alb in the same cluster and route traffic through them to the one app deployment.
+
+## Prerequirites
+
+- yc installed and configured
+- mK8S deployed in YC
+- Fill variables in `app/alb-ing.yaml` and `app/nginx-ing.yaml`
+
+## NGINX ingress + cert-manager.io Installation
+
+```bash
+kubectl create ns nginx
+kubectl create ns app
+
+# NGINX Ingress
+kubectl config set-context --current --namespace nginx
+helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+helm repo update
+helm install ingress-nginx ingress-nginx/ingress-nginx
+
+# Cert-manager
+# ! Install Cert-Manager + YC DNS Webhookubectl from MP
+# OR install Cert-Manager manually
+helm repo add jetstackubectl https://charts.jetstack.io
+helm repo update
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.11.0/cert-manager.crds.yaml
+helm install cert-manager --namespace cert-manager --version v1.11.0 jetstack/cert-manager
+
+# Configure ClusterIssuer
+kubectl apply -f -<<EOF
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt
+  namespace: cert-manager
+spec:
+  acme:
+    server: https://acme-v02.api.letsencrypt.org/directory
+    email: <EMAIL>
+    privateKeySecretRef:
+      name: letsencrypt
+    solvers:
+    - http01:
+        ingress:
+          class: nginx
+EOF
+
+kubectl get svc -n nginx # copy External IP of nginx-ingress service
+yc dns zone add-records your-zone --record  "*.nginx 60 A <EXT_IP>"
+
+kubectl config set-context --current --namespace app
+kubectl apply -f app/app.yaml
+kubectl apply -f app/nginx-ing.yaml
+
```

Comment:
please add an example of successfull output - in particular what is changed dns reconrd in zone